### PR TITLE
Add DOCS SIG members to Codeowners for /docs

### PR DESCRIPTION
**Pull request by Parsifal-M**
#### :heavy_check_mark: Checklist

- [ ] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

Hey Team! 👋 

Something we discussed in the last Docs SIG was about adding ourselves to the codeowners so we can actually see when any docs changes happen as right now we don't see anything or are notified making it hard to keep track of changes.

It was mentioned if we could do this in a round-robin way, but I have not done that before and not sure how to set it up other than doing what I have done here.

Also this is a bit of a blanket approach so happy for any guidance on how we could better set this up 😄 

FYI @awanlin, @aramissennyeydd 

The end goal is that we can keep an 👁️  on whats happening in the `/docs/*.md` 

Thanks all!

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -9,6 +9,7 @@ yarn.lock                                         @backstage/maintainers @backst
 */yarn.lock                                       @backstage/maintainers @backstage-service
 /.changeset/*.md
 /beps/0001-notifications-system                   @backstage/maintainers @backstage/notifications-maintainers
+/docs/*.md                                        @parsifal-m @awanlin @aramissennyeydd
```
Comment (vinzscam) [inline .github/CODEOWNERS:12]:
```suggestion
/docs/*.md                                        @backstage/documentation-maintainers
```

Comment (Parsifal-M) [reply .github/CODEOWNERS]: Oh, perfect 😄

**Review by awanlin — APPROVED** [review]
Thanks for this @Parsifal-M 🚀

**Review by aramissennyeydd — APPROVED** [review]

**Review by deejay1 — APPROVED** [review]

**Review by vinzscam — APPROVED** [review]

**Comment by awanlin** [comment]
Hi @deejay1, you've caught my curiosity. I've noticed that you have been approving PRs lately but most of them are without any comments or feedback in what seems a very random fashion. Could you share your end goal with these?

**Comment by deejay1** [comment]
@awanlin Well, I'm looking on what I'm currently interested in and what seems to be under my competences for reviewing. Of course I can stop doing that, if it messes up your process.

**Pull request merged**